CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ All notable changes to this project will be documented in this file.
 ### Added
 
 -   #32 : add support for `nvcc` Compiler and `cuda` language as a possible option.
+-   #48 : fix incorrect handling of imports in `cuda`.
```

Comment:
This seems to have been done badly previously so not entirely your fault but sentences should be capitalised:
```suggestion
-   #32 : Add support for `nvcc` Compiler and `cuda` language as a possible option.
-   #48 : Fix incorrect handling of imports in `cuda`.
```

Reply:
Yep, you're right m I'll fix it

Reply:
fixed 3580b26